middleware/changeLanguage.js

```diff
@@ -64,6 +65,9 @@ function updateDocs( req, res, translations ){
   // iterate over response documents
   res.data.forEach( function( doc, p ){
 
+    // update name.default to the request language (if available)
+    translateNameDefault(doc, requestLanguage);
```

Comment:
Sorry, you are using the wrong iso for our ES indexes, iso6393 is a alpha-3 code (`English = eng`) and in ES we have a alpha-2 code (`English = en`)
You can found the code here : `req.clean.lang.iso6391`

Reply:
Right, sorry about that!

Reply:
:+1: